src/Widgets/SliderWidget/SliderWidgetEditingConfig.ts

```diff
@@ -12,8 +12,8 @@ provideEditingConfig(SliderWidget, {
       description: 'Default: No',
     },
     autoplayInterval: {
-      title: 'Autoplay interval (in s)',
-      description: 'Default: 5 seconds',
+      title: 'Time between slides (in seconds)',
+      description: 'Default: 5.0 seconds',
```

Comment:
* [ ] FYI "Default: 5 seconds" was better. No unnecessary accuracy and matches what the input would show for 5.0.